docs/triage.md

```diff
@@ -0,0 +1,62 @@
+## Introduction
+
+This document will outline the process of triaging GitHub issues and provide an explanation of GitHub labels.
+
+## GitHub Labels
+
+Labels that starts with `a:` are area labels.
+Area labels help categorize issues based on their scope. Each issue should ideally have at least one area label.
+The description of the area label ends with a code that indicates the maintainer or backup maintainer of that area.
+For example, if a label description ends with `(mX,bY)`, it means person X is the maintainer, and person Y is the backup maintainer for that area.
+
+Apart from area labels, there can be other labels used to provide additional context to issues.
+
+- **Bug**: Indicates that the issue is a bug or a software defect.
+- **Enhancement**: Indicates that the issue suggests an improvement or new feature.
+- **Good First Issue**: Indicates that the issue is suitable for newcomers to contribute to the project.
+   The issue should have a good description and ideally a comment to describe how to do it
+- **Needs More Info**: Indicates that the issue lacks sufficient information and requires additional details from the reporter.
+  The maintainer for the area must remove the tag when more info is provided, or close it if too long
+
+## GitHub Assignee
+
+Here are some guidelines for GitHub assignees:
+- Assign issues to individuals who are actively working on or responsible for a particular area.
+- Avoid assigning too many issues to a single person to prevent overload.
+- Assign an issue to yourself if you plan to work on it to inform others that the issue is being addressed.
+- Assign an issue to someone else if you expect a quick action or if someone else is better suited to handle it.
+- Unassign issues that you are not actively working on to allow others to pick them up.
+
+## Issue Description
+
+A clear and concise issue description is crucial for effective communication.
+- Edit the title and description of the issue to ensure clarity and conciseness.
+- If modifications have been made to the original issue, indicate it by adding "Edited:" to the description.
```

Comment:
```suggestion
- If modifications have been made to the original issue, indicate it by adding "Edited:" to the description.
- Consider adding "acceptance criterial" so that it's easy to understand what needs to be accomplished in order to close the issue as done.
```